docs/content/docs/deep/summaryTelemetry.md

```diff
@@ -0,0 +1,363 @@
+# Summary Collection
+
+The core data structure that tracks summary attempts and corresponding results by monitoring the op log.
+
+## SummaryAckWithoutOp
+
+> Error
+
+It means that a summary ack was observed without a corresponding summary op. We only raise this event if the missing summary op's sequence number >= the initial sequence number which we loaded from.
+
+Potential causes are that a summary op was nacked then acked, double-acked, or the `summarySequenceNumber` is invalid. All cases should be recoverable, but still indicate bad behavior.
+
+- `sequenceNumber` - sequence number of the observed summary ack op.
+- `summarySequenceNumber` - sequence number of the missing summary op, as indicated by the summary ack op.
+- `initialSequenceNumber` - sequence number we initially loaded from. This is relevant since it is compared with the missing summary op sequence number to determine if we are in an error case or not.
+
+# Summary Manager
+
+> Event Prefix: `SummaryManager:`
+
+## CreatingSummarizer
```

Comment:
Should this be categorized as `Generic`? Since a lot of event are classified as `Error` or `Performance`, the rest should probably also have a category.